app/controllers/api/custom_button_sets_controller.rb

```diff
@@ -7,5 +7,12 @@ def create_resource(type, id, data)
     def edit_resource(type, id, data)
       super(type, id, data.deep_symbolize_keys)
     end
+
+    def reorder_resource(_, _, data)
+      CustomButtonSet.reorder_group_index(data["ids"])
+      action_result(true, "Button Group Reorder saved")
+    rescue => error
+      action_result(false, error.to_s)
+    end
```

Comment:
Probably not necessary but I'd feel more comfortable using the `api_action` pattern

It does little things different error reports if groups of calls are bundled together.

```suggestion
    def reorder_resource(type, id, data)
      api_action(type, id) do
        CustomButtonSet.reorder_group_index(data["ids"])
        action_result(true, "Button Group Reorder saved")
      end
    end
```